pkg/quarterdeck/mock/quarterdeck.go

```diff
@@ -189,119 +189,235 @@ func RequireAuth() HandlerOption {
 	}
 }
 
+func methodPath(method, path string) string {
+	return method + " " + path
+}
+
 func fullPath(path, param string) string {
 	if param != "" {
 		param = "/" + param
 	}
 	return path + param
 }
 
-func (s *Server) setHandler(path string, opts ...HandlerOption) {
+func (s *Server) setHandler(method, path string, opts ...HandlerOption) {
```

Comment:
Had to change the mock a bit to be able to specify different behavior for GET/POST/DELETE from the same test, which affected a lot of test files, but this should be more usable anyway.

Reply:
Makes sense to me.